.github/workflows/publish_stable_release.yml

```diff
@@ -22,21 +22,22 @@ jobs:
         with:
           token: ${{secrets.GITHUB_TOKEN}}
           tag: ${{env.VERSION}}
+          generateReleaseNotes: true
   build_and_publish_pypi:
     runs-on: ${{inputs.runner}}
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Setup Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
         with:
           python-version: "3.10"
       - name: Install Build Tools
         run: |
           python -m pip install build wheel
       - name: Build Distribution Packages
         run: |
-          python setup.py sdist bdist_wheel
+          python -m build
```

Comment:
This needs to be validated and should probably match the equivalent command in `publish_alpha_release`